### PR TITLE
Adding github to sponsors list for first time member event grant.

### PR DESCRIPTION
**Pull request by bahorn**
First step for applying for the first time member event grant.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
     "react-helmet": "^5.2.1",
     "react-icons": "^3.7.0",
     "reactstrap": "^8.0.1",
-    "styled-components": "^4.4.0"
+    "styled-components": "^4.4.0",
+    "yarn": "^1.21.1"
```
Comment (jmsv) [inline package.json]: huh? Meant to be in devdeps or added accidentally?

Comment (bahorn) [reply package.json]: Added accidentally.

Comment (jmsv) [reply package.json]: Alright - can we remove it? Then I'm happy to merge

Comment (bahorn) [reply package.json]: should be resolved now.

**Review by jmsv — APPROVED** [review]

**Pull request merged**